docs/guide/options/README.md

```diff
@@ -2,16 +2,17 @@
 
 ## setOptions
 
-| Prop                   | Type    | Default           | Description                                                           |
-|------------------------| ------- |-------------------|-----------------------------------------------------------------------|
-| dictionary             | Object  | {}                | Dictionaries for password comparison with common words/names etc.     |
-| graphs                 | Object  | {}                | Keyboard layout to check for patterns on different kind of keyboards  |
-| l33tTable              | Object  | L33tTable         | Table with matching alphabetical chars into numbers and special chars |
-| translations           | Object  | Translations keys | Translations for the feedback                                         |
-| useLevenshteinDistance | boolean | false             | Activate levenshtein                                                  |
-| levenshteinThreshold   | number  | 2                 | Threshold for levenshtein                                             |
-| l33tMaxSubstitutions   | number  | 100               | Indicated the max substituions for the l33t matcher to prevent DOS    |
-| maxLength              | number  | 256               | Defines how many characters of the password are checked               |
+| Prop                   | Type    | Default           | Description                                                          |
+|------------------------| ------- |-------------------|----------------------------------------------------------------------|
+| dictionary             | Object  | {}                | Dictionaries for password comparison with common words/names etc.    |
+| graphs                 | Object  | {}                | Keyboard layout to check for patterns on different kind of keyboards |
+| l33tTable              | Object  | L33tTable         | Table with matching alphabetical chars into numbers and special chars|
+| translations           | Object  | Translations keys | Translations for the feedback                                        |
+| useLevenshteinDistance | boolean | false             | Activate levenshtein                                                 |
+| levenshteinThreshold   | number  | 2                 | Threshold for levenshtein                                            |
+| l33tMaxSubstitutions   | number  | 100               | Indicated the max substituions for the l33t matcher to prevent DOS   |
+| maxLength              | Object  | 256               | Defines how many characters of the password are checked              |
+| timeEstimationValues   | number  | {}                | Define the values to calculate the scoring and attack times          |
```

Comment:
Looks like you changed the wrong line here.

`maxLength` should be a `number`
`timeEstimationValues` should be an `Object`